### ISSUE TITLE
Duplicate Sinks.ApplicationInsights namespace?

### ISSUE THREAD

**Issue by cmeeren** [serilog-contrib/serilog-sinks-applicationinsights#99]
Tried implementing a new telemetry converter and noticed that `ITelemetryConverter` and its implementations are in the namespace 

`Serilog.Sinks.ApplicationInsights.Sinks.ApplicationInsights.TelemetryConverters`

Should `Sinks.ApplicationInsights.` really occur twice in that path?

**Comment by thomgre** [comment]
If you look at this line in src the namespace seems to be correct:
https://github.com/serilog/serilog-sinks-applicationinsights/blob/master/src/Serilog.Sinks.ApplicationInsights/TelemetryConverter.cs#L15
```
using Serilog.Sinks.ApplicationInsights.Sinks.ApplicationInsights.TelemetryConverters;
```

**Comment by cmeeren** [comment]
I see I was a bit unclear. I wasn't wondering whether it was actually the case. I was wondering whether it is an intended namespace or a bug, and if perhaps the duplicate part of the namespace should be removed.

**Comment by nblumhardt** [comment]
Fixed in 4.0.0-dev 👍
